README.md

```diff
@@ -4,23 +4,21 @@ A really great Dota 2 game mode built by [really great people](/contributors.md)
 # Installation
 This addon uses the [Dota 2 Addon Manager](https://github.com/chrisinajar/dota2-addon-manager).
 
-For detailed instructions, check [here](docs/install.md)
+For installation instructions click [here](docs/install.md).
+
+For testing instructions click [here](docs/testing.md).
 
 # Contributing
-Check out the Discord!
+Check out our [Discord](https://discord.gg/WNFBB4d)!
```

Comment:
why remove the url variable? also i'm not sure if we want this on the front page yet.  Also this has nothing to do with gulp.